packages/jest-reporters/src/notify_reporter.ts

```diff
@@ -123,7 +126,7 @@ export default class NotifyReporter extends BaseReporter {
             message,
             title,
           },
-          (err, _, metadata) => {
+          (err: any, _: any, metadata: any) => {
```

Comment:
```suggestion
          (err: Error | null, _: any, metadata: any) => {
```

can we type metadata?

---

Or, shouldn't this be part of `node-notifier`'s typings?